### ISSUE TITLE
sort_by should be a JSON pointer

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-discovery#480]
- The value of sort_by should be a JSON pointer so it can refer to any field, including nested ones
- However JSON pointers need to start with a /, generally
- Example does not do this...

**Comment by farshidtz** [comment]
I agree that the current spec isn't clear on what to do with nested fields and support choosing JSON Pointer as **the only notation** in Discovery v2. However, we need to make sure such a change doesn't add implementation burden for servers which use relational or semantic DBs.
